track_and_trace/asset_client/src/views/list_assets.js

```diff
@@ -33,14 +33,17 @@ const AssetList = {
     vnode.state.currentPage = 0
 
     const refresh = () => {
-      api.get('records?recordType=asset').then((records) => {
-        vnode.state.records = records
+      m.request({
+        method: 'GET',
+        url: '/grid/record'
+      })
+      .then((result) => {
+        vnode.state.records = result
```

Comment:
You should probably filter the result here so that it only shows records that have schema === 'asset'. Otherwise, when we have "fish" records it will also show on this list.

Reply:
Good call. Updated.